parm/post_avblflds_raphrrr.xml

```diff
@@ -6156,5 +6156,52 @@
        <scale>4.0</scale>
     </param>
 
+    <param>
```

Comment:
@SamuelTrahanNOAA This file is only used for ops. hrrr/rap only. There is no need to incorporate new IFI fields.

Reply:
reverted change